packages/maskbook/src/plugins/ITO/assets/formatDate.ts

```diff
@@ -14,7 +14,7 @@ export function datetimeISOString(value: Date): string {
     if (value && value instanceof Date && !isNaN(value.valueOf())) {
         const pad = (value: number) => value.toString(10).padStart(2, '0')
         const date = [value.getFullYear(), value.getMonth() + 1, value.getDate()]
-        const time = [value.getHours(), value.getMinutes(), value.getSeconds()]
```

Comment:
Do we have the requirement to accurate the date to the second?

Reply:
> @lelenei I find only a small area is clickable at which picker would popup by clicking. Do you encounter the same problem?
> 
> <img alt="" width="400" src="https://user-images.githubusercontent.com/63733714/117258794-ee7a3c80-ae7f-11eb-92e6-bd69f4b6d601.png">

I deleted the picture of the date selection

Reply:
This is used to input the display date